src/Command/DispatchCommand.php

```diff
@@ -141,12 +141,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
     private function updateRepositories(Package $package, array $projectConfig): void
     {
         $repositoryName = $this->getRepositoryName($package);
+        $lastVersion = $this->getLastPackageVersion($package);
```

Comment:
Can we catch here and set `http://sonata-project.org`, as default?

And I would prefer to remove the trailing `/` from the url

Reply:
I do `'homepage' => $latestVersion->getHomepage() ?: 'https://sonata-project.org',`

isn't it enough ?